docs/reference/decomposers.rst

```diff
@@ -170,3 +178,39 @@ arbitrary variables.
 
     >>> print(state1.subproblem)
     BinaryQuadraticModel({'z': -2.0, 'y': 0.0}, {('z', 'y'): 4.0}, 0.0, Vartype.BINARY)
+
+SublatticeDecomposer
+--------------------
+
+This example creates a 5x5 square ferromagnetic lattice problem,
+and builds the 3x3 problem located at the center of the square.
+The initial state is set to all spin up.
+Only the variable (2,2) is not adjacent to the boundary, other
+variables pick up a linear bias of 1 or 2 due to the boundary condition.
+Origin embedding keys determine the subproblem created, in this
```

Comment:
"Origin embedding keys" do you mean "Keys of `the `origin_embeddings`` dict"?

Reply:
Also corrected some pep8 formatting errors in subsequent block